contracts/contracts/RequestManager.sol

```diff
@@ -66,53 +83,99 @@ contract RequestManager is Ownable {
         bytes32 fillId
     );
 
+    /// Emitted when staked native tokens tied to a claim have been withdrawn.
+    ///
+    /// This can only happen when the claim has been resolved and the caller
+    /// of :sol:func:`withdraw` is allowed to withdraw their stake.
     event ClaimWithdrawn(
         uint256 claimId,
         uint256 indexed requestId,
         address claimReceiver
     );
 
     // Constants
+
+    /// The minimum amount of source chain's native token that the claimer needs to
+    /// provide when making a claim, as well in each round of the challenge game.
     uint256 public claimStake;
+
+    /// The period for which the claim is valid.
     uint256 public claimPeriod;
+
+    /// The period by which the termination time of a claim is extended after each
+    /// round of the challenge game. This period should allow enough time for the
+    /// other parties to counter-challenge.
+    ///
```

Comment:
There is a missing piece. The termination time is only extended by it if current terminition time is shorter than `challengePeriodExtension`

Reply:
Added a note on that.